src/System.CommandLine/Binding/Binder.cs

```diff
@@ -7,15 +7,50 @@ namespace System.CommandLine.Binding
 {
     internal static class Binder
     {
-        internal static bool IsMatch(this string parameterName, string alias) =>
-            string.Equals(alias?.RemovePrefix()
-                              .Replace("-", "")
-                              .Replace("|", "or"),
-                          parameterName,
-                          StringComparison.OrdinalIgnoreCase);
+        internal static bool IsMatch(this string parameterName, string alias)
```

Comment:
This was misidentified as a change because it was a expression bodied member and it was changed into block body method. That's why I always put the arrow in the next line if it's not a one-liner.

The API was already defined with non-nullable strings but was accepting null strings. Removed the null checks.